src/mixins/common-mixin.ts

```diff
@@ -16,9 +16,9 @@ export default class CommonMixin extends Vue {
    * Scrolls the window to the top of the specified element.
    * @param element the element to scroll to the top of
    */
-  async scrollToTop (element: any): Promise<void> {
+  scrollToTop (element: Element): void {
     // don't call window.scrollTo during Vitest tests because jsdom doesn't implement it
-    if (!this.isVitestRunning) await element.scrollIntoView({ behavior: 'smooth' })
+    if (!this.isVitestRunning) element.scrollIntoView({ behavior: 'smooth' })
```

Comment:
`Element.scrollIntoView()` is not async.

Thus, `scrollToTop()` doesn't need to be async, nor its callers, etc.